services/mac_manager/src/main/java/com/futurewei/alcor/macmanager/service/implement/MacRedisServiceImpl.java

```diff
@@ -93,7 +97,8 @@ public String releaseMacState(String macAddress) throws Exception {
             macPoolRedisRepository.addItem(macAddress);
             macRedisRepository.deleteItem(macAddress);
         }
-        return macAddress;
+        return macState.getMacAddress();
+        //return new String ("{mac_address: " + macAddress+"}");
```

Comment:
Remember to remove commented codes later.

Reply:
I removed it.